react/features/speaker-stats/components/web/SpeakerStats.js

```diff
@@ -76,7 +76,7 @@ class SpeakerStats extends Component<Props> {
             <Dialog
                 cancelKey = 'dialog.close'
                 submitDisabled = { true }
-                titleKey = 'speakerStats.speakerStats'
+                titleKey = 'speakerStats.title'
```

Comment:
Please undo this. While inconsistent it messes translations up because they all need to be updated, and just for changing the title, in an internal fine, doesn't look like a good reason IMHO.